daphne_worker_test/tests/e2e.rs

```diff
@@ -1529,7 +1529,7 @@ async fn e2e_helper_admin_add_task(version: DapVersion) {
         "vdaf_verify_key": "y4e6alnJMQ0MZTvdJRJx5Q"
     });
 
-    let url = Url::parse("http://127.0.0.1:8788/task").unwrap();
+    let url = Url::parse("http://helper:8080/task").unwrap();
```

Comment:
For local testing, it would be necessary to add `helper` as an alias for `localhost`. Would you mind noting this in daphne_worker_test/README.md?